### PR TITLE
MTM-47173 | updating microservices images...

### PR DESCRIPTION
**Pull request by manasesjesus**
...and readme

### CODE REVIEW

**Comment by BeateRixen** [comment]
@manasesjesus The images have the wrong size. They must be taken with a browser width of 1440 px. If you have done so, ( I guess so since the width is exactly double size) then the issue might be, that for Macs you have to half the size of the image due to the retina display.

**Comment by BeateRixen** [comment]
The image showing the microservices still does not show the microservice which is mentioned in the surrounding text, does it? Not sure about the other one.

**Comment by manasesjesus** [comment]
@BeateRixen both images are correct. The surrounded text is “Subscribe” and it’s a button. 
From the hello world tutorial:

Once the ZIP file has been uploaded successfully, you will see a new microservice application created.
>> here goes admin-microservice-deployed.png image showing Ecosystem and all microservices

….

**Subscribe to the microservice**

In the Administration application, navigate to Ecosystem > Microservices. Locate your microservice application and click it to open its details. On the top right, click **Subscribe**.
>> here goes admin-microservice-subscribe.png

**Comment by BeateRixen** [comment]
> @BeateRixen both images are correct. The surrounded text is “Subscribe” and it’s a button. From the hello world tutorial:
> 
> Once the ZIP file has been uploaded successfully, you will see a new microservice application created.
> 
> > > here goes admin-microservice-deployed.png image showing Ecosystem and all microservices
> 
> ….
> 
> **Subscribe to the microservice**
> 
> In the Administration application, navigate to Ecosystem > Microservices. Locate your microservice application and click it to open its details. On the top right, click **Subscribe**.
> 
> > > here goes admin-microservice-subscribe.png

First image:
The surrounding text is talking about the hello world microservice: 
Once the ZIP file has been uploaded successfully, you will see a new microservice application created.
The image should show "my" microservice which I created in the tutorial.
Second image:
Locate your microservice application and click it to open its details.
The image should also show "my" microservice which I created in the tutorial.

File: README.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ layout: root # don't change
 weight: 90 # order the section in the section dropdown in ascending order
 ---
 ```
-
+<!-- TODO: fix broken link -->
 Grab the icon classes in the [styleguide](http://styleguide.cumulocity.com/icons/)
```
Comment (manasesjesus) [inline README.md:54]: @BeateRixen this needs to be updated. I don't know who knows the correct link.

Comment (manasesjesus) [inline README.md:54]:
```suggestion
Grab the icon classes in the [Styleguide](https://styleguide.cumulocity.com/#/icons/cumulocity)
```

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**